### DISCUSSION TITLE
Serial Output

### DISCUSSION THREAD

**Discussion by j45p41** [h2zero/n-able-Arduino#25] [Q&A] [answered]
Hi Thank you for a great library, which solves almost all my problems with other libraries!

I am using the following config in my platformio.io:

[env:nrf52840_dk_adafruit]
platform = https://github.com/h2zero/platform-n-able.git#1.0.0
board = nrf52840_dk
framework = arduino
; board_bootloader = adafruit
monitor_speed = 115200
upload_protocol = jlink
debug_tool = jlink
lib_deps =
    h2zero/NimBLE-Arduino@^1.4.0

however I can't see any serial output.

**Comment by h2zero** [comment]
You're welcome. I'm not sure what could be causing this issue for you, it's working for my 52833 DK.

**Reply by j45p41** [reply, marked as answer]
Hi,

thank you for your response.

I have figured it out - firstly after doing some reading I found that the second USB port (on side of board) of Serial - this was probably obvious but I just figured it!

Also for others trying to do the same thing. Here is my platformio.ini that works:

```
[env:nrf52840_dk_adafruit]
platform = https://github.com/h2zero/platform-n-able.git#1.0.0
board = nrf52840_dk
framework = arduino
board_bootloader = adafruit
monitor_speed = 115200
upload_protocol = jlink
debug_tool = jlink

build_flags =
    ${env.build_flags}
    -DCORE_DEBUG_LEVEL=5

lib_deps =
    h2zero/NimBLE-Arduino@^1.4.0
```


Also let me take this opportunity to say that this library has been a lifesaver for my project and thank you for making it!

**Reply by h2zero** [reply]
Great! Glad you got it working!

You're very welcome 😊
